src/lib.rs

```diff
@@ -43,34 +43,91 @@ impl Sequence {
         }
     }
 
-    /// Seeks to the given sample offset.
-    pub fn seek(&mut self, sample: u64) {
+    /// Seeks to the given sample offset. Starts at dimension 0 of the given sample.
+    pub fn seek_to(&mut self, sample: u64) {
         self.sample = sample;
         self.dimension = 0;
     }
+    
+    /// Seeks a relative offset of samples. Starts at dimension 0 of the sample which is seeked to.
+    /// Performs a wrapping addition.
+    pub fn seek(&mut self, offset: u64) {
```

Comment:
Maybe could use an enum for relative/absolute seeking like the std::io one, thoughts? Too overkill?

Reply:
Either could work, doesn't really matter to me